pkg/operation/botanist/component/kubeapiserver/networkpolicies.go

```diff
@@ -140,6 +141,14 @@ func (k *kubeAPIServer) reconcileNetworkPolicyAllowKubeAPIServer(ctx context.Con
 					From: []networkingv1.NetworkPolicyPeer{
 						// Allow all other Pods in the Seed cluster to access it.
 						{PodSelector: &metav1.LabelSelector{}},
+						// Allow all istio-ingress Pods in the Seed cluster to access it.
+						{
+							// we don't want to modify existing labels on the istio namespace
+							NamespaceSelector: &metav1.LabelSelector{},
+							PodSelector: &metav1.LabelSelector{MatchLabels: map[string]string{
+								v1beta1constants.LabelApp: gardenletconfigv1alpha1.DefaultIngressGatewayAppLabelValue,
```

Comment:
Does this work as well, if additional exposure classes are configured in gardenlet?
Maybe we should add and use a common label, e.g. `istio=ingressgateway`.

Reply:
This is exactly the label that all exposure class deployments of istio have in common.

@dkistner: Feel free to confirm/deny.

Reply:
If I remember correctly the only label that all ingress gateway pods (default and exposure class ones) have is `app: istio-ingressgateway`.

Reply:
(which is what I used here for that reason)